lib/foreman_maintain/settings.rb

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+module ForemanMaintain
+  class Settings
+    def self.load_from_file(file_dir)
+      file_path = File.join(file_dir, 'foreman_maintain.yml')
```

Comment:
Alternatively, we could let the `ForemanMaintain.setup` to accept a strnig, in which case it would assume it was a patch to yaml and perform the logic there, which might look a bit nicer than in executable